pkg/logs/interface.go

```diff
@@ -3,7 +3,16 @@ package logs
 import "io"
 
 type Client interface {
-	// DevModeLogs gets logs for the provided component name and namespace. A component could have multiple pods and
-	// containers running on the cluster. It returns a slice of maps where container name is the key and its logs are the value
+	// DevModeLogs gets logs for the Dev mode containers of the provided component name and namespace. A component could
+	// have multiple pods and containers running on the cluster. It returns a slice of maps where container name is the
+	// key and its logs are the value.
```

Comment:
It is not clear what is the array for. A map  for each pod?

Reply:
Yes.